go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/louy/terraform-provider-uptimerobot
+
+go 1.12
+
+require (
+	github.com/hashicorp/hcl v0.0.0-20180404174102-ef8a98b0bbce // indirect
```

Comment:
Thanks @nlamirault! Can you explain why this dependency is needed? I can't find it in other providers like [aws](https://github.com/terraform-providers/terraform-provider-aws/blob/master/go.mod) and [mysql](https://github.com/terraform-providers/terraform-provider-mysql/blob/master/go.mod)